### PR TITLE
Increase timeouts of url requests

### PR DESCRIPTION
**Pull request by glesys-andreas**
Some API queries takes up to 8-10 seconds to complete and Ansible has a default timeout of 10 seconds on url queries, so sometimes it fails.

### CODE REVIEW

**Comment by hejhansson** [comment]
maybe rephrase this commit: "Ignore response from API, it is not always reliable, we get this data later".

**Comment by glesys-andreas** [comment]
@hejhansson Yeah, i'm going to squash my development commits before merging.
